Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb

```diff
@@ -6,13 +6,17 @@
    "source": [
```

Comment:
<p>is this NB viewer who moved the assignments or are the assignments below the variable? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>